h/static/scripts/login-forms/routes.ts

```diff
@@ -8,4 +8,5 @@ export const routes = {
   login: '/login',
   forgotPassword: '/forgot-password',
   signup: '/signup',
+  loginWithORCID: '/oidc/login/orcid',
```

Comment:
Apparently this is how we're doing routes for the new login and signup pages. I think these should really be passed from the backend to the frontend via the `js-config`.

Reply:
I agree. https://github.com/hypothesis/h/pull/9699/files#diff-1f483dadf62736d9ba68775e2282f18efd28895d1e1376d5ee6d1a5d8213ea75 added a new `routes` section to the config which we could extend with all the required routes.